lib/OpenQA/CacheService/Response/Info.pm

```diff
@@ -25,10 +25,18 @@ sub available_workers {
     return $data->{active_workers} != 0 || $data->{inactive_workers} != 0;
 }
 
+sub inactive_jobs_exceeded {
+    my $self = shift;
+    return undef if $self->max_inactive_jobs < 0;
+    return undef unless my $data = $self->data;
+    return $data->{inactive_jobs} > $self->max_inactive_jobs;
+}
+
 sub availability_error {
     my $self = shift;
     return $self->error if $self->has_error;
     return 'No workers active in the cache service' unless $self->available_workers;
+    return 'Already cache service jobs piling up in queue' if $self->inactive_jobs_exceeded;
```

Comment:
How about phrasing it more objectively and mentioning the limit? To avoid the "did the config take effect" question

```suggestion
    return 'Cache service queue is already full ($max_inactive_jobs)' if $self->inactive_jobs_exceeded;
```